### ISSUE TITLE
in stratagem list view it wont  scroll down when I select something off the screen

### ISSUE THREAD

**Issue by Cmonster820** [byjokese/stratagem-hero#2]
![Screenshot 2024-03-12 7 44 32 AM](https://github.com/byjokese/stratagem-hero/assets/72608260/01064f5e-8f69-4e0a-b25a-44cab00fbba7)

**Comment by Cmonster820** [comment]
zooming out chrome fixed the issue, but I am leaving this open because it probably isn't intended to have this problem in the first place

**Comment by Cmonster820** [comment]
nevermind, it did not, it just made more stuff visible

**Comment by byjokese** [comment]
Yeah, I may need to design that for small screen, I do have a massive screen, feels really different.

On my 16:9
![image](https://github.com/byjokese/stratagem-hero/assets/5951126/effdbf92-39f8-482c-9607-607e2ddf376b)

On my Ultrawide, 21:9
![image](https://github.com/byjokese/stratagem-hero/assets/5951126/de868fd2-abb6-42d1-be1e-f8f0c80c90de)

I did it responsive, but not that much. Will look at it.

Will see if I can update it, for the Controller support update.🤞

**Comment by byjokese** [comment]
I'm trying to stay on the arcade style for all the UI, but it really makes it more difficult, than just a normal UI that you can use the mouse...

@Cmonster820 your screen seems to be a standard Laptop HiDPI screen. Will check for that size. Didn't want to get to tablet sizes as arcades don't fit well on mobiles, but maybe I can make it a bit more responsive.

**Comment by Cmonster820** [comment]
thanks
